web/libs/datamanager/src/components/MainView/GridView/GridView.jsx

```diff
@@ -12,56 +12,87 @@ import { FF_GRID_PREVIEW, FF_LOPS_E_3, isFF } from "../../../utils/feature-flags
 import { SkeletonLoader } from "../../Common/SkeletonLoader";
 import { GridViewContext, GridViewProvider } from "./GridPreview";
 import "./GridView.scss";
+import { clsx } from "clsx";
+clsx;
+import { groupBy } from "../../../utils/utils";
 
-const GridHeader = observer(({ row, selected }) => {
+const TEXT_ONLY_CELL_HEIGHT = 250;
```

Comment:
this can be moved as a separate value to `TextDataGroup`, something like `TextDataGroup.heightWithoutImages = 250`